ansible/playbooks/teleop_deploy.yml

```diff
@@ -112,6 +114,7 @@
       become: yes
 
   roles:
+    - { role: products/common/get-system-variables }
```

Comment:
Is this needed on the nuc?

Reply:
but is there a system variable on nuc for desktop? I remember we create a desktop folder at some point. Have you checked it on nuc?

Reply:
So after testing this branch on server_and_nuc (with nuc plugged into server) the branch executed fine with no errors. After execution I SSH'd onto the nuc and ran the xdg-user-dir DESKTOP which returned this screenshot (I'll pm it to you). So I believe this works as intended on the NUC and should be left there incase we ever need future system variables like these.